dump/plugins/__init__.py

```diff
@@ -9,7 +9,10 @@
 
 # import child classes automatically
 for (module_loader, name, ispkg) in pkgutil.iter_modules([pkg_dir]):
-    importlib.import_module('.' + name, __package__)
+    try:
+        importlib.import_module('.' + name, __package__)
+    except ImportError as e:
+        pass
```

Comment:
>pass [](http://example.com/codeflow?start=8&length=4)

swallow exception blindly is dangerous. Do you really need it?

Reply:
This is used to print the modules available for the dump utility :
```
admin@sonic:~$ dump state --show
Module              Identifier
------------------  ---------------------
acl_rule            acl_rule_name
acl_table           acl_table_name
copp                trap_id
evpn                Remote VNI
fdb                 Vlan:fdb_entry
interface           intf_name
port                port_name
portchannel         portchannel_name
portchannel_member  portchannel_member
route               destination_network
vlan                vlan_name
vlan_member         vlan_member_name
vxlan_tunnel        vxlan_tunnel_name
vxlan_tunnel_map    vxlan_tunnel_map_name
admin@sonic:~$
```
I did not want to add anything else here so that on platforms where the newly added dash modules are not present there is no indication of errro

Reply:
I understand your goal, however this swallowing exception may hide more issue than you expected. Could you fix the underlying throw, or further check specific module throws ImportError?

Reply:
Added check for specific module, please check